client/java/src/main/java/com/linecorp/centraldogma/internal/client/AbstractWatcher.java

```diff
@@ -258,6 +262,12 @@ private void doWatch(int numAttemptsSoFar) {
                          logger.info("{}/{}{} does not exist yet; trying again",
                                      projectName, repositoryName, pathPattern);
                          logged = true;
+                         if (errorOnEntryNotFound) {
+                             latest = null;
+                             if (!initialValueFuture.isDone()) {
+                                 initialValueFuture.completeExceptionally(thrown);
+                             }
```

Comment:
I think we should do `retrun null` here so that we don't call `scheduleWatch` again.

Reply:
Is it okay don't call `close()` method?

Reply:
Yes, we should call `close()`. Thanks!